privacyidea/cli/pitokenjanitor/utils/findtokens.py

```diff
@@ -696,3 +695,28 @@ def remove_tokeninfo(ctx, tokeninfo_key):
             token_obj.del_tokeninfo(tokeninfo_key)
             token_obj.save()
             click.echo(f"Removed tokeninfo '{tokeninfo_key}' for token {token_obj.token.serial}")
+
+@findtokens.command('export_for_privacyidea')
+@click.option('--file', required=False, type=click.Path(writable=True),
+              help='The file to export the tokens to. Defaults to "exported_tokens.txt".')
+@click.pass_context
+def export_token_for_privacyidea(ctx, file):
```

Comment:
Can we rewrite this as a format option for the `export` command? This way we can also add the file option to the `export` command. And the default output could then be stdout (We can still require a file for the `privacyidea` format but the user should provide it)